File: 2000_Reverse_Prefix_of_Word.py
import string

class Solution:
    def reversePrefix(self, word: str, ch: str) -> str:
        decider = -1
        for i in range(len(word)):
            if (word[i] == ch):
                decider = i
                break

        ans = ""
        if (decider == -1):
            ans = word
        else: 
            ans = word[i::-1] + word[i + 1:]

        return ans

File: 2000_Reverse_Prefix_of_Word.cpp
#include <iostream>
#include <algorithm>
using namespace std;

string reversePrefix(string word, char ch) {
    for (int i = 0; i < word.size(); i++) {
        if (word[i] == ch) {
            reverse(word.begin(), word.begin() + i + 1);
            break;
        }
    }        

    return word;
}

int main() {
    return 0;
}